bin/node/runtime/src/lib.rs

```diff
@@ -686,7 +694,14 @@ pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, Call, SignedExtra>;
 /// Executive: handles dispatch to the various modules.
-pub type Executive = frame_executive::Executive<Runtime, Block, frame_system::ChainContext<Runtime>, Runtime, AllModules>;
+pub type Executive = frame_executive::Executive<
+	Runtime,
+	Block,
+	frame_system::ChainContext<Runtime>,
+	Runtime,
+	AllModules,
+	(),
```

Comment:
Are there use cases for customizing the top-level (transaction-level) dispatcher?
1. If not, I'd remove this.
2. If yes, I'd rather see an `enum DefaultDispatcher {}` type that implements the dispatcher instead of `()`. While `()` works for associated types, here it lacks the context of what we are actually setting (You have to go to generic parameters and hope it has a nice readable name there).

Reply:
There might be cases where someone want to provide another `RootDispatcher`. This is a far less intrusive change for a runtime implementer than modifying the `frame_system` which is the default `RootDispatcher`. Therefore I argue that it provides some benefit.

That said, I removed this parameter because we already specify the `RootDispatcher` through the `frame_system::trait::RootDispatcher`. No need for a redundant parameter.